apps/openmw/mwmechanics/enchanting.cpp

```diff
@@ -224,6 +280,9 @@ namespace MWMechanics
 
         float priceMultipler = MWBase::Environment::get().getWorld()->getStore().get<ESM::GameSetting>().find ("fEnchantmentValueMult")->mValue.getFloat();
         int price = MWBase::Environment::get().getMechanicsManager()->getBarterOffer(mEnchanter, static_cast<int>(getEnchantPoints() * priceMultipler), true);
+        int count = getEnchantItemsCount();
+        if (count > 1)
+            price *= count * 0.125f;
```

Comment:
The 1/8 multiplier seems to be arbitrary. Maybe put it into a setting?

Reply:
Wouldn't this make a count of 8 equal to 1 in price, and 2 to 7 be cheaper than 1?

Reply:
I guess that this multiplier is in place because enchanting multiple arrows would have been outrageously expensive otherwise.

Reply:
It may be better to scale the price based on the item type then, rather than the count. Otherwise, enchanting 1 arrow will be much more expensive than enchanting 2. And if there were to ever be a QoL enhancement of enchanting multiple objects at once (at an equal cost to doing one at a time, just more efficiently), this count check would break it.

Reply:
Oh right, that seems to be wrong.